### ISSUE TITLE
Page titles (WCAG 2.4.2)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1236]
**Request:** The intent of this Success Criterion is to help users find content and orient themselves within it by ensuring that each Web page has a descriptive title. Titles identify the current location without requiring users to read or interpret page content.

**Issues:**
1. https://slidewiki.org/user/vivien.zschammer/recommendations 
- The page does not have a meaningful page title.

2. The page titles should match the selected page language.
- https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/edit 
- https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/markdownEdit 
- https://slidewiki.org/search?keywords=accessibility&sort=score 
- https://slidewiki.org/user/vivien.zschammer/recommendations 
- https://slidewiki.org/addDeck 
- https://slidewiki.org/contactus 
- https://slidewiki.org/deck/127411/test/deck/127411 
- https://slidewiki.org/accessibility 
- https://slidewiki.org/user/vivien.zschammer/settings/profile
- https://slidewiki.org/user/vivien.zschammer/stats

**Comment by aoelen** [comment]
I am currently working on this, but I remove the `easy fix` label. Unfortunately this requires a lot of changes because the titles are set when the translation context is not yet available 😅

**Comment by aoelen** [comment]
The linked pull request fixes this issue. As you can see in the code, it required really a lot of changes to support translatable page titles
